### ISSUE TITLE
y_svar missing sqlite and mysql.

### ISSUE THREAD

**Issue by Ahmad45123** [pawn-lang/YSI-Includes#46]
By looking in y_svar, There was a planned feature to add mysql and sqlite support.
However its missing in the y_svar folder.

**Comment by Y-Less** [comment]
The actual plan was a little more involved than that.  y_users, y_uvar, and y_svar are all based on y_ini.  The idea was to develop a consistent API so that the backend could be swapped for any of them without them knowing which was in use.

**Comment by Misiur** [comment]
Yeah. I don't have much time to do this at the time, but the main issues are: 
1. Only few people are aware how the y_users data is compressed
2. MySQL introduces another layer of complexity (whole SQL language and RDBMS specific quirks), also it's callback based (tho y_ini is as well...). So, making the backend seemless regardless of type might be an issue.

**Comment by Y-Less** [comment]
There's no reason it should be compressed in the same way by other backends.
